src/services/config-store.js

```diff
@@ -58,6 +54,10 @@ class ConfigStore {
     if (options.onlyCrudModule) {
       this.logger.warn('The use of onlyCrudModule is deprecated. Please remove the option in Forest initialization.');
     }
+
+    if (options.modelsDir) {
+      this.logger.warn('The use of modelsDir is deprecated. Please remove the option in the Forest initialization.');
```

Comment:
(sorry for the noise: I thought about this this morning during the standup and did not want to interrupt) Is it **deprecated** or **unsupported**? It seems it's not supported anymore, so it's not deprecated. It's removed and we have a warning.